ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/handshake/ecies/EncryptedMessage.java

```diff
@@ -57,19 +58,19 @@ public static Bytes decryptMsg(final Bytes msgBytes, final SECP256K1.PrivateKey
 
     // Perform the decryption.
     final ECIESEncryptionEngine decryptor =
-        ECIESEncryptionEngine.forDecryption(ourKey, ephPubKey, iv);
+        ECIESEncryptionEngine.forDecryption(nodeKey, ephPubKey, iv);
     return decryptor.decrypt(encrypted);
   }
 
   /**
    * Decrypts the ciphertext using our private key.
    *
    * @param msgBytes The ciphertext.
-   * @param ourKey Our private key.
+   * @param nodeKey Abstraction of this nodes private key & associated crypto operations
    * @return The plaintext.
    * @throws InvalidCipherTextException Thrown if decryption failed.
    */
-  public static Bytes decryptMsgEIP8(final Bytes msgBytes, final SECP256K1.PrivateKey ourKey)
+  public static Bytes decryptMsgEIP8(final Bytes msgBytes, final NodeKey nodeKey)
```

Comment:
just wondering, should we name argument as 'ourNodeKey' ?

Reply:
I'd prefer not to. For 2 reasons:
Thinking "nodeKey" is explicit enough to indicate ownership (on some levels).
Secondly, I'd hate for this class to imply the owner of the key which is doing the decryption - i.e. it simply needs a key to do the decryption - ownership of the key makes no difference as to the operation being conducted.